apis/scheduling/v1alpha1/types.go

```diff
@@ -113,6 +115,8 @@ const (
 	PodGroupLabel = "pod-group." + scheduling.GroupName
 )
 
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
Same here. Remove k8s tags for kubebuilder owned bits.

(applies elsewhere)

Reply:
as I mentioned before, the deepcopy runtime.objects are needed to be created hence we need this annotation. Found it used in the plugins implementations.